docs/advanced/composition.md

```diff
@@ -3,20 +3,28 @@ You can compose multiple stores together using class inheritance. This is quite
 
 ```TS
 @State({
-  name: 'zoo'
+  name: 'zoo',
+  defaults: {
+    type: null 
+  }
 })
 class ZooState {
   @Action(Eat)
-  eat() {}
+  eat(ctx: StateContext) {
```

Comment:
I think we should use destructing (`{ setState }`) in the docs.

Reply:
There was a desion a while back to remove the destructuring because it is not as familiar to most devs as a normal parameter. See PR #320. We should still use `ctx`.